src/core.ts

```diff
@@ -774,8 +795,9 @@ export class TemplateInstance {
           partIndex++;
         } else if (nodeIndex === part.index) {
           if (part.type === 'node') {
-            this._parts.push(
-                this.processor.handleTextExpression(node, this._getTemplate));
+            const part = this.processor.handleTextExpression(this._getTemplate);
+            part.insertAfterNode(node);
```

Comment:
Could we make `handleTextExpression` responsible for this?

Reply:
I like `handleTextExpression` not knowing about the assumption that we're attaching to a marker node like this.